azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -339,6 +344,7 @@ func resourceArmKeyVaultCertificateRead(d *schema.ResourceData, meta interface{}
 	// Computed
 	d.Set("version", id.Version)
 	d.Set("secret_id", cert.Sid)
+	d.Set("certificate_data", string(*cert.Cer))
```

Comment:
I was wondering about that. It seemed like a certificate should always be set, but I'll put in a check.

Reply:
that's _generally_ true - however the Azure API's return the state of a resource when it was last modified, rather than as it is right now (as such, it's possible it could be nil)